gimp/pictonode-gimp-plugin/GIMP 2.99/scripts/Linux/install-flatpak.sh

```diff
@@ -6,13 +6,22 @@ then
     python3 ../GtkNodes/setup_gtknodes.py
 fi
 
+if [ "$1" == "" ]
+then
+   plugins_dir=~/.var/app/org.gimp.GIMP/config/GIMP/2.99/plug-ins
+else
+   plugins_dir=$1
+   mkdir $plugins_dir
+   chmod 777 $plugins_dir
```

Comment:
Is 777 really necessary here?

Reply:
It's necessary for an external-from-flatpak parent plugin directory, whereas the default uses a parent plugin directory the flatpak application already has permissions for. Looked a little bit into flatpak extensions today, it seems feasible and thinking its something we should do after we get through our jira tickets.